### PR TITLE
Makefile for SciPy 1.0 paper

### PR DESCRIPTION
**Pull request by person142**
Working on gh-8. Still need to hook things up to a CI, but it seems worthwhile to let people comment on the makefile first.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+**/*~
+
+**/*.aux
+**/*.fdb_latexmk
+**/*.fls
+**/*.log
+**/*.pdf
```
Comment (tylerjereddy) [inline .gitignore]: Good idea. There's a pretty well thought out `.gitignore` in the [official github gitignore repo for LaTex](https://github.com/github/gitignore/blob/master/TeX.gitignore) which I tend to pull from progressively as a work builds up in complexity / intermediate files generated.

Comment (tylerjereddy) [reply .gitignore]: We can probably safely add some ignore tags for popular editors too, but that burden doesn't all have to fall on this PR -- I'm sure one of us will eventually start seeing swap files and just patch the ignore file.

Comment (tylerjereddy) [reply .gitignore]: Have we decided on the citation approach yet? When we do that, there will be more stuff to add here as well.

**Comment by rgommers** [comment]
Makefile works for me, thanks @person142

File: scipy-1.0/Makefile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+TEX = $(wildcard *.tex)
+
```
Comment (perimosocordiae) [inline scipy-1.0/Makefile]: Worth adding: `.PHONY: paper clean`

File: .gitignore (hunk 1 above)
Comment (tylerjereddy) [reply .gitignore]: Yeah, something like that.

**Comment by person142** [comment]
Ok, I got CircleCI working in my fork and just pushed those changes here. I think an owner will have to turn on builds for this repository.

**Comment by person142** [comment]
This was the result in my fork:

https://26-74767083-gh.circle-artifacts.com/0/papers/artifacts/paper.pdf

**Comment by person142** [comment]
To keep things rolling, bump on this point:

> an owner will have to turn on builds for this repository.

**Comment by rgommers** [comment]
@person142 done!

It said in the instructions:

    create a folder named .circleci and add a file config.yml (so that the filepath be in
    .circleci/config.yml).

but this should work as well

**Comment by rgommers** [comment]
Okay that worked, can see the pdf under artifacts.

**Comment by rgommers** [comment]
Okay works, so merging. Follow up on content of `.gitignore` etc can be tweaked later.

**Comment by person142** [comment]
Thanks @rgommers.

**Pull request merged**